<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Tx_Transmission</class>
 <widget class="QDialog" name="Dlg_Tx_Transmission">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableView" name="tableView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="v_Layout_Commandes">
       <item>
        <widget class="QPushButton" name="button_Ajouter">
         <property name="text">
          <string>&amp;Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Supprimer">
         <property name="text">
          <string>&amp;Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Selectionner">
         <property name="text">
          <string>Selectionner</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Fermer">
         <property name="text">
          <string>&amp;Fermer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="gb_edit_champs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="title">
        <string>Ajouter un élement :</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="h_Layout_Designation">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Désignation :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Designation"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="v_Layout_Ajout">
          <item>
           <widget class="QPushButton" name="button_Annuler">
            <property name="text">
             <string>&amp;Annuler</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_Valider">
            <property name="text">
             <string>&amp;Valider</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
